doc/sphinx/proofs/writing-proofs/equality.rst

```diff
@@ -485,7 +485,7 @@ which reduction engine to use.  See :ref:`type-cast`.)  For example:
 
       Abort.
 
-.. tacn:: lazy {? @reductions } @simple_occurrences
+.. tacn:: {? weak } lazy {? @reductions } @simple_occurrences
```

Comment:
Even with your changes from #17507, this won't work, because the names should come after the signature, and should include `cbv` (in the middle, with the current implementation of #17507).

Reply:
Answers are $5.  Correct answers are $10.  Dumb looks are still free :-)  Please DWIM.